packages/manager/apps/dedicated/client/app/ip/ip/ip-ip.html

```diff
@@ -4,191 +4,201 @@
+                            data-ng-pluralize
+                            data-count="alertsCount()"
+                            data-when="{
+                                        '0': ('ip_alerts_none' | translate) + ' :',
+                                        'one': ('ip_alerts_1' | translate) + ' :',
+                                        'other': ('ip_alerts_other' | translate: { count: {} }) + ' :'
+                                    }"
+                        >
+                        </span>
+                        <span
+                            data-ng-if="singleService()"
+                            data-ng-pluralize
+                            data-count="alertsCount()"
+                            data-when="{
+                                    '0': ('ip_alerts_service_none' | translate) + ' :',
+                                    'one': ('ip_alerts_service_1' | translate) + ' :',
+                                    'other': ('ip_alerts_service_other' | translate: { count: {} }) + ' :'
+                                }"
+                        >
+                        </span>
+                    </p>
 
-                        <p data-ng-if="alerts.mitigation.length">
-                            <span
-                                class="d-block"
-                                data-ng-pluralize
-                                data-count="alerts.mitigation.length"
-                                data-when="{
-                                      '0': ('ip_alerts_mitigation' | translate),
-                                      'one': ('ip_alerts_mitigation_1' | translate),
-                                      'other': ('ip_alerts_mitigation_other' | translate: { count: {} })
-                                  }"
-                            >
-                            </span>
-                            <button
-                                class="btn btn-link"
-                                type="button"
-                                data-ng-if="!singleService()"
-                                data-ng-repeat="ip in alerts.mitigation"
-                                data-ng-click="navigateToService(ip.ip.routedTo.serviceName)"
-                                data-ng-bind="ip.ip.routedTo.serviceName"
-                            ></button>
-                        </p>
-                    </div>
+                    <p data-ng-if="alerts.spam.length">
+                        <span
+                            class="d-block"
+                            data-ng-pluralize
+                            data-count="alerts.spam.length"
+                            data-when="{
+                                    '0': ('ip_alerts_spam' | translate),
+                                    'one': ('ip_alerts_spam_1' | translate),
+                                    'other': ('ip_alerts_spam_other' | translate: { count: {} })
+                                }"
+                        >
```

Comment:
I am not sure that case 0 will be valid, because we have as root condition ```data-ng-if="alerts.spam.length"```.
Right ?

Reply:
No modifications done here. It's showing in the changes section because of the change in the layout structure.